jest.conf/testUtils.js

```diff
@@ -29,3 +22,56 @@ export const testAfterResize = testFunction => {
   };
   animationFrameId = requestAnimationFrame(assertAfterResize);
 };
+
+export async function renderComponent(page, component, props) {
+  await page.evaluate(
+    ({ component, props }) => {
+      window.postMessage(
+        {
+          type: 'RENDER_COMPONENT',
+          component: component,
+          props: props,
+        },
+        '*'
+      );
+    },
+    { component, props }
+  );
+  await page.waitForSelector('#testing-playground');
+
+  const isComponentRendered = await page.evaluate(() => {
+    const testing_playground = document.querySelector('#testing-playground');
+    return testing_playground && testing_playground.children.length > 0;
+  });
+
+  if (!isComponentRendered) {
+    // eslint-disable-next-line no-console
+    console.error('Component did not render in the testing playground');
+  }
+}
+
+export async function takeSnapshot(page, name) {
+  if (process.env.TEST_TYPE == 'visual') {
+    await percySnapshot(page, name);
+  } else {
+    // eslint-disable-next-line no-console
+    console.log(`Skipping Percy snapshot: ${name}`);
```

Comment:
Now that we are running this method just in visual tests, I think we can remove this else block.

Reply:
Sure :+1: